tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -897,7 +897,6 @@ public function callAPISuccessGetValue($entity, $params, $type = NULL) {
   public function callAPISuccessGetSingle($entity, $params, $checkAgainst = NULL) {
     $params += array(
       'version' => $this->_apiversion,
-      'debug' => 1,
```

Comment:
because passing this attribute add xdebug properties in the array, which later impacts ```checkArrayEquals``` where it only concern about matching the values of the array, the reason why allowing this attribute mismatches with expected array and leads to test failures.